src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const result = {};
+  const strings = sourceString.split(';').map((str) => str.trim());
+
+  strings.forEach((string) => {
+    const [key, ...values] = string.split(':');
+    const value = values.join('').trim();
```

Comment:
The use of the spread operator '...' to handle multiple values is unnecessary here since you expect only one value after the colon. You can directly access the second element after the split using index [1].

Reply:
considering the given example, it is possible to have multiple values after the colon (e.g., all .2s ease-in-out)